### DISCUSSION TITLE
什么时候框架支持huatuo热更新？

### DISCUSSION THREAD

**Discussion by sue602** [JasonXuDeveloper/JEngine#353] [General]
如题。

**Comment by JasonXuDeveloper** [comment]
抱歉哈，刚看到这个讨论。
JEngine目前很多功能是针对ILRuntime做的，例如内存级别的热更代码加密、ClassBind、MonoBehaviour生命周期优化等，这些功能大部分都不兼容HybridClr，所以暂时没有换HybridClr的想法。

**Comment by JasonXuDeveloper** [comment]
已支持
